README.md

```diff
@@ -3,17 +3,17 @@
 Hello :wave:, we are team AbstractSingletonProxyFactoryBean,
 and this is our submission for [PyWeek 31](https://pyweek.org/).
 
-### Installation
+## Installation
 In your terminal of choice, run
 ```commandline
-$ git clone https://github.com/SavagePastaMan/pyweek-2021.git
-$ cd directory_name
+$ git clone https://github.com/SavagePastaMan/flawless.git
+$ cd flawless
 $ python -m pip install -r requirements.txt
 $ python -m game
 ```
 Depending on your platform, you may need to use `python3` or `py`.
```

Comment:
just incase, can we specify 3.9

Reply:
iirc the rules say it should be latest version, but i can add that, yeah